### ISSUE TITLE
Update link in Github Config

### ISSUE THREAD

**Issue by madiedgar** [legesher/tree-sitter-legesher-python#42]
**Update Github Config Link**
The link in the `.github/config.yml` is incomplete. Please update the link to include the `.html` extension at the end of the url. 

It should look like this:

``` yml
We'd love to celebrate you and your contribution to the community. Let us say "Thank You" again, by sending you a [special gift](https://legesher.io/dreamteam.html) only given to special individuals like you who have chosen to give back.
```

**Comment by kevintham** [comment]
Hi, I would like to work on this!

**Comment by madiedgar** [comment]
That would be amazing! Thanks @kevintham, I'll assign you to this!

**Comment by madiedgar** [comment]
This was fixed in #43 & #44
